CONTRIBUTING.md

```diff
@@ -74,6 +74,32 @@ environment variable to disable webpack optimizations.
 env DEBUG=true yarn start:prod
 ```
 
+## Authentication in your local Force app
+
+Authentication in Force is handled through OAuth, with [Gravity](https://github.com/artsy/gravity) authenticating the user and redirecting back to Force. For security reasons, the `localhost` origin [is forbidden as a redirect URL by Gravity in the staging environment](https://github.com/artsy/gravity/blob/543373d7d4413f5c8b1c8f84f73b2a592c00cba2/app/models/util/url_validation.rb#L23). This means that when running Force locally at `http://localhost:5000`, you won't be able to sign up or log in.
+
+### Browse locally while logged in
+
+If you need to run Force locally while logged in but don't need authentication to run locally, you can log into `http://staging.artsy.net`. The user information will propagate to your local browsing session.
```

Comment:
I am pretty sure this is true, but I haven't undone my `local.artsy.net` setup to verify.

Reply:
This section feels a bit confusing, is it necessary?

Reply:
I left it out originally, but as I was writing the section about `local.artsy.net` I thought "how is it possible that I did not run into this problem until 1.5 years in?" I thought it was worth acknowledging that you don't *need* to run as `local.artsy.net` unless you need the auth flow to fully work.

Though now that you mention that the user can just refresh the page, it's very possible that's what I've been doing all along and just don't remember it...so maybe the content of this section isn't even accurate.